src/mixins/breakpointsMixin.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  computed: {
+    smAndUp() {
+      return this.$vuetify.display.smAndUp;
+    },
+    smAndDown() {
+      return this.$vuetify.display.smAndDown;
+    },
+    isMobile() {
```

Comment:
Just picky me: some functions have an "is..." and some are just named after the vuetify breakpoint codes. I suggest to stick to consistent naming. And keeping the vuetify nomenclature makes sense.

(In fact I stumbled over "isXs" and wondered if that's a thing in vuetify... 😉 )